netaddr.go

```diff
@@ -632,6 +632,23 @@ func (ip IP) IsLinkLocalMulticast() bool {
 	return false // zero value
 }
 
+// IsGlobalUnicast reports whether ip is a global unicast address.
```

Comment:
for IPv6, is this only 2000::/3 or does it also include ULAs? It should be clear from docs.

Reply:
Should I use the stdlib comment as a starting point?

> The identification of global unicast addresses uses address type identification as defined in RFC 1122, RFC 4632 and RFC 4291 with the exception of IPv4 directed broadcast addresses. It returns true even if ip is in IPv4 private address space or local IPv6 unicast address space.

Reply:
That looks like classic Mikioh style documentation, referencing a sea of RFCs numbers & sections. That's fine to include, but I'd also like something more readable with higher-level descriptions/summaries/examples. I shouldn't need to go read a dozen RFCs to understand such a simple-looking function.

Reply:
@bradfitz updated, feedback welcome. It almost seems like this might be better as a bulleted list, but what do you think?